app/src/main/res/layout/activity_album_tag_editor.xml

```diff
@@ -139,7 +139,7 @@
     <View
         android:layout_width="match_parent"
         android:layout_height="?android:attr/actionBarSize"
```

Comment:
could you replace the height with:
android:layout_height="@dimen/toolbar_scrim_height"

this is already used in the flat player and to me it seems like the best of both word, dark enough with the fifty_percent_black_overlay and big enough to not be too noticeable.

Reply:
Done